src/com/chrome/apis/polymer.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.CLASS({
+  package: 'foam.u2.md',
+  name: 'StackView',
+  extends: 'foam.u2.BasicStackView',
+  methods: [
+    function initE() {
+      this.cssClass('flex').cssClass('layout').cssClass('vertical');
+      this.SUPER();
+    }
+  ]
+});
+
+foam.CLASS({
+  package: 'foam.u2.md',
+  name: 'Toolbar',
+  extends: 'foam.u2.Element',
+  properties: [
+    {
+      name: 'title'
+    },
+    {
+      name: 'leftActions_',
+      factory: function() {
+        return [];
+      }
+    },
+    {
+      name: 'rightActions_',
+      factory: function() {
+        return [];
+      }
+    },
+    [ 'nodeName', 'paper-toolbar' ]
+  ],
+
+  methods: [
+    function initE() {
+      this.add(this.slot('leftActions_'));
```

Comment:
It makes the style checker angry. It thinks `foo_$` is fine, but `fooBar_$` is mixing `camelCase` with `snake_case`.

We can patch the style enforcer in time, but for now I was trying to keep these files style-clean.